app/imports/templates/components/newBid.js

```diff
@@ -18,60 +18,69 @@ Template['components_newBid'].onRendered(function() {
     });
   }
 
-  this.autorun(() => {
-
-    let name = Session.get('searched');
-
-    // The goal here is to obscure the names we actually want
-    function randomName() {
+  // The goal here is to obscure the names we actually want
+  function randomName() {
+    if (typeof knownNames !== "undefined") {
```

Comment:
Does this provide a weaker set of names while the list is still loading? Could we not make this asynchronous and wait until it's loaded instead?